frontend/src/pages/Candidates.tsx

```diff
@@ -1,3 +1,10 @@
+import { PathwayTimeline } from "../components/index.ts";
+
 export function Candidates() {
-  return <h2>Candidates</h2>;
+  return (
+    <>
+      <h2>Candidates</h2>
+      <PathwayTimeline path={1} progress={2} />
```

Comment:
Maybe keep this and the import out since we are not going to use the timeline in the Candidates page. Everything else looks great